### PR TITLE
External webhook

### PR DESCRIPTION
**Pull request by phpai**

### CODE REVIEW

File: scripts/pipeline/publish-docker-images.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  docker push $TAG
+fi
+
+if [ -n "$TRAVIS_TAG" ];
+then
+  echo "Enter TAG Path"
+  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  docker tag "$TAG" "$TAG_LATEST"
+  docker push $TAG
```
Comment (kevinbader) [inline scripts/pipeline/publish-docker-images.sh]: quotes missing

File: scripts/pipeline/publish-docker-images.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+echo "$PRIVATE_REGISTRY_PASSWORD" | docker login -u "$PRIVATE_REGISTRY_USERNAME" --password-stdin "$PRIVATE_REGISTRY"
+docker tag "$TAG" "$TAG_BUILD_PRIVATE"
+echo "Pushing [private]/trubudget/$PROJECT_NAME:$TRAVIS_BUILD_ID"
+docker push "$TAG_BUILD_PRIVATE" >/dev/null 2>&1
+
+if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_EVENT_TYPE" = "push" ];
+then
+  echo "Enter Master Path"
+  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  docker push $TAG
```
Comment (kevinbader) [inline scripts/pipeline/publish-docker-images.sh]: quotes missing

Comment (phpai) [reply scripts/pipeline/publish-docker-images.sh]: done

**Comment by phpai** [comment]
@kevinbader kindly re-review please :) thx

**Pull request merged**